domains/qloha.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "Domain name for Sora OS",
+    "repo": "https://github.com/qloha/Sora-OS",
+    "owner": {
+        "username": "qloha",
+        "discord": "qloha"
+    },
+    "record": {
+        "CNAME": "39zrzhpn.up.railway.app"
```

Comment:
404 site when trying to access your site. Please provide a preview or something similar.

Reply:
@DEV-DIBSTER I have it in the pull request? the cname record is the thing that said to put in, not the one thats generated from railway
## Website Link/Preview
<!-- Please provide a link or preview of your website below. If you can't make the website visible, then an image of the website is also fine! -->
https://sora-os.up.railway.app
As I said earlier, this will be getting updates as I add more things. Sorry for the wrong link earlier.